src/components/Scene/BouncingPeep.tsx

```diff
@@ -0,0 +1,49 @@
+import useStyles from './BouncingPeep.styles'
+import { greetings } from '../../config/text'
+
+interface Props {
+	greeting: string
```

Comment:
Can `greeting` be state managed in the component instead? Either a default value passed in or call the random function for initial state?